tests/ui/test_login.py

```diff
@@ -1,11 +1,25 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
+from seleniumlogin import force_login
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.django_db
+@pytest.mark.skipif('localhost' not in os.getenv('PYTEST_BASE_URL'),
+                    reason='No force login for dev, prod testing')
```

Comment:
Another problem with ```live_server```. I could not get it to login using fxa accounts. something seems to be blocking it but it would never authenticate. fxa account login on olympia.dev reroutes to olympia.dev, however trying to route back to the ```live_server.url``` would never work. All this does it force a login using the credentials made by fxapom and and the supercreate to login.